content/changes.md

```diff
@@ -8,6 +8,38 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## April 30, 2024 
+
+### Improvements 
+
+- **Make quiz cutoffs more consistent** 
```

Comment:
I'd say this is a bit of a borderline content tweak that happens often behind the scenes. I'm not totally against including it, but if we were to include it, I'd keep the details out and say something like "We lowered the passing scores on some of our quizzes to improve scoring accuracy".

Another thing about evidence engine is that it's only available to a small-ish subset of customers with signals enabled, if I understand correctly, so that also makes it a bit less critical to announce and I think pushes it below the threshold for release note inclusion.